packages/cactus-cmd-gui-app/README.md

```diff
@@ -0,0 +1,48 @@
+# `@hyperledger/cactus-cmd-gui-app`
+
+This component allows viewing ledger data in Supabase. The data is fed to supabase by persistence plugins for each ledgers.
+
+## Summary
+
+- [Remarks](#remarks)
+- [Getting Started](#getting-started)
+- [Contributing](#contributing)
+- [License](#license)
+- [Acknowledgments](#acknowledgments)
+
+## Remarks
+
+- Plugin requires running Supabase and persistence plugins in order to properly view ledger data.
+- Currently, only ethereum based ledgers are supported.
+
+## Getting Started
+
+Clone the git repository on your local machine. Follow these instructions that will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+In the root of the project, execute the command to install and build the dependencies. It will also build this GUI front-end component:
+
+```sh
+yarn run build
+```
+
```

Comment:
@ervvkb The build command will only perform the build but won't install the dependencies AFAICT so I'd recommend having the configure script or a `yarn install` here as well.

The easiest way to double check if your documentation is accurate is by running the steps one by one yourself and then seeing if/where it fails. (boring but often yields great results in terms of figuring out that some commands were missing that newcomers will immediately get stuck on as they are trying to use the code)